public/i18n/en/translation.json

```diff
@@ -920,6 +927,24 @@
       "destination_url_tooltip": "This path specifies the location to which incoming requests will be forwarded. Example: 'http://destination-server:port/path'",
       "allowed_networks_tooltip": "Access permitted only from the following IPv4/IPV6 networks"
     },
+    "qos": {
+      "title_short": "QoS",
+      "title": "Quality of Service",
+      "description": "Lorem ipsum dolor sit amet, consectetur adipiscing elit. Fusce vitae velit arcu.",
+      "no_interface_found": "No interface found",
+      "add_qos_interface": "Add QoS interface",
+      "edit_qos_interface": "Edit QoS interface",
+      "interface": "Interface",
+      "download_speed": "Download speed",
+      "download_speed_tooltip": "Lorem ipsum dolor sit amet, consectetur adipiscing elit. Fusce vitae velit arcu.",
+      "upload_speed": "Upload speed",
+      "upload_speed_tooltip": "Lorem ipsum dolor sit amet, consectetur adipiscing elit. Fusce vitae velit arcu.",
+      "status": "Status",
+      "delete_qos_interface": "Delete QoS interface",
+      "delete_qos_interface_message": "Are you sure you want to delete '{interface}'?",
+      "choose_interface": "Choose interface",
+      "add_interface": "Add interface"
+    },
```

Comment:
@cotosso, @gsanchietti, any lorem ipsum replacement?

Reply:
On it!

Reply:
Done. @filippocarletti can you check please?

Reply:
I just added my suggestion